internal/cli/atlas/maintenance/update.go

```diff
@@ -66,7 +67,10 @@ func UpdateBuilder() *cobra.Command {
 	opts := &UpdateOpts{}
 	cmd := &cobra.Command{
 		Use:   "update",
-		Short: "Update the maintenance window.",
+		Short: "Update the maintenance window for your project.",
```

Comment:
Suggestion: In other PRs, I've seen usage of _modify_ instead of _update_ in the description here. Please change, if needed, to be consistent with other CLI pages.